src/components/Person/Person.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Person, Sex } from '../../types';
+
+type Props = {
+  person: Person;
+  hasMother: Person | null;
+  hasFather: Person | null;
+};
+
+export const PersonLink: React.FC<Props> = ({
+  person,
+  hasMother,
+  hasFather,
+}) => {
+  const {
+    name, slug, sex, born, died, motherName, fatherName,
+  } = person;
+
+  const personLink = classNames({
```

Comment:
The link will not be stored in this variable, the class for the link will be defined there
```suggestion
  const personLinkClass = classNames({
```